pages/quests/quest1.mdx

```diff
@@ -94,7 +94,12 @@ LOGS = LOGS or {}
 Define a handler for adding tokens to the price feed.
 
 <br />
+
 <i>**6.1: Handler Definition**</i>
+<br />
+The `Handlers.add` function adds a new handler or updates an existing handler by name. <br />
+Next up is Handlers.utils library which provides two functions that are common matching patterns and one function that is a common handle function.
+The `Handlers.utils.hasMatchingTag` returns a function that requires a message argument, so you can drop this into any pattern argument on the Handlers module. The function compares the Tag Name and Value, if they are equal then it invokes the handle function.
```

Comment:
- ~returns a function~ is a function 
- Tag Name & Value should be in `code` syntax